Sloth.Web/Sloth.Web.csproj

```diff
@@ -22,6 +22,8 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+
+    <PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.16" />
```

Comment:
Was there some problem that was solved by adding this?

Reply:
No, VS added it when I tried to use the razor page generator